### DISCUSSION TITLE
Tag groups separation

### DISCUSSION THREAD

**Discussion by vorandrew** [dailydotdev/daily#547] [Feature Requests]
I need ability to read different tag groups in separate feed... For example "backend" and "frontend". I don't want to read mixed

**Comment by idoshamun** [comment]
Hi @santacruz123,

Thanks for the suggestion. It's not planned at the moment to support this use case but I understand your point so I'll raise it in our next planning meeting. Thanks a lot :)

**Reply by rranzolin** [reply]
I think this would be a great UX improvement, it's very confusing to read all categories grouped, in my opinion it would be great to search by category I want in addition to the ones I have in My Feed

**Reply by idoshamun** [reply]
We are now rethinking this whole process so stay tuned :)

**Reply by youssefKadaouiAbbassi** [reply]
Hi @idoshamun ! What if we could create multiple tabs, name them, and choose the tags from the different categories we want on them. This will come back to having the possibility of cloning the "My feed" tab multiple times and renaming it.

And by the way, keep the great job guys 🔥!

**Reply by idoshamun** [reply]
This can definitely work but is still not prioritized from our side. I'll keep you posted 😬
